### DISCUSSION TITLE
Configure string on Tool Settings

### DISCUSSION THREAD

**Discussion by BhaveshAnandpara-bentley** [iTwin/itwinjs-core#7354] [Q&A]
We want to replace this message with an empty string or a different message. Is there any way to do it ?

![Image (1)](https://github.com/user-attachments/assets/50091a6c-4bbe-43b9-bd01-456e861a8447)

**Comment by ben-polinsky** [comment]
Hi,

  Looking into this, I think changes in [AppUI](https://github.com/iTwin/appui/) are necessary.  The [message](https://github.com/iTwin/appui/blob/30e6cc13085855ec20ca55181dd9b18ccb8128d2/ui/appui-react/src/appui-react/widget-panels/ToolSettings.tsx#L33) consists of a few localized strings, one of which is used in other contexts.
  
  @GerardasB any thoughts?

**Reply by GerardasB** [reply]
> Hi,
> 
> Looking into this, I think changes in [AppUI](https://github.com/iTwin/appui/) are necessary. The [message](https://github.com/iTwin/appui/blob/30e6cc13085855ec20ca55181dd9b18ccb8128d2/ui/appui-react/src/appui-react/widget-panels/ToolSettings.tsx#L33) consists of a few localized strings, one of which is used in other contexts.
> 
> @GerardasB any thoughts?

I agree, since the message is formatted from multiple localized strings and a tool name (which is a localized string as well) it is a bit tricky, i.e. this is the closest I got by overriding the localized strings as a workaround (in a real application, you should probably re-map to other localized keys to keep the localization support or simply provide different localization files): https://www.itwinjs.org/sandboxes/GerardasB/localization-override

@BhaveshAnandpara-bentley please create a feature request here https://github.com/iTwin/appui/issues/new/choose I think we could add additional configuration to https://www.itwinjs.org/reference/appui-react/frontstage/frontstageconfig/#toolsettings

**Reply by BhaveshAnandpara-bentley** [reply]
Hey @GerardasB @ben-polinsky ,
I have added a feature request [here](https://github.com/iTwin/appui/issues/1120). Let me know if this is fine or you will require more information.
